app/components/value-wheel/value-wheel.tsx

```diff
@@ -1,12 +1,25 @@
-import React, { useState } from "react";
+import { useState } from "react";
+
 import { Donut } from "./donut";
-import * as styles from "./value-wheel.module.scss";
 
-export const ValueWheel = () => {
+export interface ValueProposition {
+  id: string;
+  textColor: string;
+  text: string;
+  content: string;
+  color: string;
+}
+
+interface ValueWheelProps {
+  title: string;
+  valuePropositions: ValueProposition[];
+}
+export const ValueWheel = (props: ValueWheelProps) => {
   const [active, setActive] = useState(-1);
   return (
-    <div className={styles.container}>
+    <div className="w-5/6 max-w-xl my-0 mx-auto">
```

Comment:
🗒️  Blir sånn cirka ish the samme som det som var definert it `value-wheel.module.scss` 🤷 
```css
.container {
  width: 80%;
  max-width: 600px;
  margin: 0 auto;
}
```